### PR TITLE
RULEAPI-598 Provide a way to name a language subdirectory correctly …

### PR DESCRIPTION
**Pull request by amelie-renard-sonarsource**
…(when specializing a rule)

### CODE REVIEW

File: supported_languages.adoc [hunk 1]
```diff
@@ -0,0 +1 @@
+`abap`, `apex`, `cfamily`, `cobol`, `csharp`, `css`, `flex`, `go`, `html`, `java`, `javascript`, `kotlin`, `php`, `pli`, `plsql`, `python`, `rpg`, `ruby`, `rust`, `scala`, `solidity`, `swift`, `tsql`, `vb6`, `vbnet` or `xml`
```
Comment (arseniy-sonar) [inline supported_languages.adoc:1]:
```suggestion
`abap`, `apex`, `cfamily`, `cobol`, `csharp`, `css`, `flex`, `go`, `html`, `java`, `javascript`, `kotlin`, `php`, `pli`, `plsql`, `python`, `rpg`, `ruby`, `rust`, `scala`, `solidity`, `swift`, `tsql`, `vb6`, `vbnet`, or `xml`
```

File: validate_asciidoc.sh [hunk 2]
```diff
@@ -17,10 +17,15 @@ do
     fi
   else
     #validate asciidoc
-    for language in $dir/*/
+	supportedLanguages="abap apex cfamily cobol csharp css flex go html java javascript kotlin php pli plsql python rpg ruby rust scala solidity swift tsql vb6 vbnet xml"
```
Comment (arseniy-sonar) [inline validate_asciidoc.sh:20]:
```suggestion
	supportedLanguages=$(sed 's/ or//' supported_languages.adoc | tr -d '`,')
```
To reduce duplication where we can :)

**Review by arseniy-sonar — CHANGES_REQUESTED** [review]
Looks good. A couple of things I think can still be improved:

File: validate_asciidoc.sh (hunk 2 above)
Comment (amelie-renard-sonarsource) [reply validate_asciidoc.sh:20]: Great! I did not know how to do that...

**Pull request merged**